### ISSUE TITLE
Integration with plugin "JS Support Ticket "

### ISSUE THREAD

**Issue by herrvigg** [qtranslate/qtranslate-xt#373]
<a href="https://github.com/FrancescoASessa"><img src="https://avatars1.githubusercontent.com/u/18053767?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [FrancescoASessa](https://github.com/FrancescoASessa)**
_Thursday Apr 14, 2016 at 14:05 GMT_
_Originally opened as https://github.com/qTranslate-Team/qtranslate-x/issues/373_

----

How can I integrate your plugin with JS Support Ticket.
Basically I would need to be able to enter the FAQ into several languages how to articles
![image](https://cloud.githubusercontent.com/assets/18053767/14530765/a912a550-025a-11e6-8255-f70ac71bdb38.png)
![image](https://cloud.githubusercontent.com/assets/18053767/14530768/ae2d67be-025a-11e6-9435-991a64b84d0a.png)
Thanks

**Comment by herrvigg** [comment]
<a href="https://github.com/johnclause"><img src="https://avatars2.githubusercontent.com/u/5491361?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [johnclause](https://github.com/johnclause)**
_Wednesday Apr 27, 2016 at 03:18 GMT_

----

Does this help: https://qtranslatexteam.wordpress.com/integration/ ?
[FAQ](https://qtranslatexteam.wordpress.com/faq/)?

**Comment by herrvigg** [comment]
<a href="https://github.com/jcnavamorffe"><img src="https://avatars3.githubusercontent.com/u/11263629?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [jcnavamorffe](https://github.com/jcnavamorffe)**
_Wednesday Dec 07, 2016 at 15:19 GMT_

----

i instaled vc-mega-menu, a plugins for Visual Composer, also, the compatibility plugins for VC Bakery for qtranslate-x, and work fine, but with that plugin (vc-mega-menu) no. :(

help Please
